engine/rpcserver.go

```diff
@@ -1303,6 +1303,20 @@ func (s *RPCServer) WithdrawalEventByID(_ context.Context, r *gctrpc.WithdrawalE
 // WithdrawalEventsByExchange returns previous withdrawal request details by exchange
 func (s *RPCServer) WithdrawalEventsByExchange(_ context.Context, r *gctrpc.WithdrawalEventsByExchangeRequest) (*gctrpc.WithdrawalEventsByExchangeResponse, error) {
 	if !s.Config.Database.Enabled {
+		if r.Id == "" {
```

Comment:
I think I'd prefer `gctrpc.WithdrawalEventsByExchangeRequest` to contain a property which selects whether to go via API or database here. Essentially: 
```
.\gctcli.exe withdrawalrequesthistory byexchange --exchange=binance --currency=USDT --source=api
```
Where `source` is either `database` or `api` and that determines which endpoint to select here. This means that I can still retrieve withdrawalhistory if I want if I have the database enabled.

What do you think?

Reply:
Hello! :) I'm not sure if this won't confuse users, for example if someone sends request "--currency=USDT --source=database" and expects to get only USDT results, but that wouldn't be so now. In that case need to change appropriate functions to filter DB results by currency, right?

Reply:
Fair enough! I didn't notice the lack of currency filter in that function! So nevermind. That's a bit more work to achieve and can always be done later. tACK!